docs/theming-flatpak.rst

```diff
@@ -0,0 +1,56 @@
+Theming Flatpak
+===============
+
+Flatpak apps default to Adwaita as a theme, as they cannot use the system theme. This happens because all flatpaks are sandboxed and do not have the ability to read data files or libraries in ``/usr`` (where system themes are typically located). This is done so that the format is portable across a wide range of distributions in a reproducible environment, but has the downside of not being able to pick up themes. The solution to this was to package GTK themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as extensions to the Gnome runtime.
+
+Installing themes
+-----------------
+
+The current themes are packaged in the flathub repository which you can add (if it's not already added) by running::
+
+  $ flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
Should mentione for Qt:

```
flatpak remote-add kdeapps https://distribute.kde.org/kdeapps.flatpakrepo

flatpak search kstyle
flatpak install kdeapps org.kde.KStyle.Adwaita//5.9

flatpak search platformtheme
flatpak install kdeapps org.kde.PlatformTheme.QGnomePlatform//5.9
```

Reply:
Ah, I didn't know there was a kdeapps repository. Also, I presume the installation of kstyle and platformtheme from said repository is necessary for the Qt theming to work?

Reply:
Yes. At some point I'd love for these to be auto-installed too but I'm not super familiar with Qt.